client/gutenberg/extensions/plugins/atavist/plugin.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
Is this code necessery? I though that Jetpack would determine when this code gets loaded.

Reply:
This file is actually unrelated to Jetpack. The Map block is going to be included in Jetpack but this (and other similar blocks) are also being included in a stand-alone plugin, and this `.php` file relates to the plugin only.